static/sass/styles.scss

```diff
@@ -190,7 +190,7 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
 // Makes side-navigation fit with the custom top-navigation of c.com
 .p-side-navigation {
   &--raw-html.is-sticky {
-    z-index: 102;
+    z-index: 5; // make sure it's higher than 2, that is used by some components (like the rules)
```

Comment:
Is this change required? If you inspect the page the navigation has `z-index: 99`, so I am not sure this would have an effect
![image](https://github.com/user-attachments/assets/f855d788-e8e1-423b-a002-c9df4d9aee83)